### PR TITLE
Scale on hover for refCard and dataSourcesCard

### PR DESCRIPTION
**Pull request by snehankekre**
## 📚 Context

The refCard and dataSourcesCard components don't scale on hover, unlike the Tile module.

## 🧠 Description of Changes

- Scale on hover for refCard and dataSourcesCard

**Revised:**

https://user-images.githubusercontent.com/20672874/188625332-a26c381a-8a86-4487-8ca7-e10a984740b4.mov

## 💥 Impact

Size:

- [x] Small 
- [ ] Not small

### CODE REVIEW

File: components/blocks/refCard.module.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .Container {
-  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-70;
+  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-90;
+}
+
+.Container:hover {
+  transform: scale(1.05);
```
Comment (imjuangarcia) [inline components/blocks/refCard.module.css:6]: When hovering over reference cards, the text becomes a bit blurry:

Screen Shot 2022-09-06 at 1 11 06 PM

I'm not sure if it's the `scale` or the `opacity` (or both) causing this, but I wonder if that's desirable, especially on API reference cards which feature quite a bit of text (and also code snippets)

Comment (snehankekre) [reply components/blocks/refCard.module.css:6]: Oh good catch! I missed the blurring without my glasses on 😅 We definitely do not want any text blurring, especially on API reference cards

File: components/blocks/refCard.module.css [hunk 2]
```diff
@@ -1,5 +1,9 @@
 .Container {
-  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-70;
+  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-90;
```
[the remark this answers is not in the document]
Comment (snehankekre) [reply components/blocks/refCard.module.css:2]: I've confirmed the blurring is removed when `hover:opacity-90` is removed

File: components/blocks/refCard.module.css (hunk 1 above)
Comment (snehankekre) [reply components/blocks/refCard.module.css:6]: `hover:opacity-90` was causing the blurring. It is removed when `hover:opacity-90` is removed.

File: components/blocks/dataSourcesCard.module.css [hunk 3]
```diff
@@ -1,5 +1,9 @@
 .Container {
-  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-70;
+  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-90;
```
[the remark this answers is not in the document]
Comment (snehankekre) [reply components/blocks/dataSourcesCard.module.css:2]: I wasn't able to figure out why `scale-105` isn't working. Removing `hover:scale-105 hover:border-b-gray-30 hover:opacity-90` and relying on `transform: scale` seems to resolve most issues that were brought up

**Review by imjuangarcia — APPROVED** [review]
Thanks for these last updates! I think it looks good for now, although we might want to do a pass later on to standardize our hover effects a bit more.

But other than that, I think this is shippable ⛵

**Pull request merged**